Services/OperationService/Commands/GetBotsCommand.cs

```diff
@@ -0,0 +1,43 @@
+﻿using DTO;
+using DTO.BrokerRequests;
+using DTO.RestRequests;
+using Interfaces;
+using Kernel;
+using MassTransit;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace OperationService.Commands
+{
+    public class GetBotsCommand : ICommand<Guid, List<BotInfoResponse>>
+    {
+        private readonly IRequestClient<InternalGetBotsRequest> client;
+        private readonly ILogger<GetBotsCommand> logger;
+
+        public GetBotsCommand
+            ([FromServices]IRequestClient<InternalGetBotsRequest> client,
+            [FromServices] ILogger<GetBotsCommand> logger)
+        {
+            this.client = client;
+            this.logger = logger;
+
+        }
+
+        private async Task<List<BotInfoResponse>> GetBotsByUserId(InternalGetBotsRequest request)
```

Comment:
Don't really see the reason of using this response class (again saying it, yes)

Reply:
fixed